migration/migration.toml

```diff
@@ -297,10 +301,26 @@ rewrite="Math.max"
 match="Math.min_float"
 rewrite="Math.min"
 
+[math-a-g]
+match="Math.abs_int"
+rewrite="Math.Int.abs"
+
+[math-a-h]
+match="Math.abs"
+rewrite="Math.abs"
```

Comment:
Unless I misunderstand how this works, this doesn't seem to do anything?

Reply:
Oops, you are correct. Changed to `Math.abs_float`